Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -752,6 +749,23 @@ public TriangularMovingAverage TRIMA(Symbol symbol, int period, Resolution? reso
         }
 
         /// <summary>
+        /// Creates a new T3MovingAverage indicator.
+        /// </summary>
+        /// <param name="symbol">The symbol whose T3 we want</param>
+        /// <param name="period">The period over which to compute the T3</param>
+        /// <param name="volumeFactor">The volume factor to be used for the T3</param>
+        /// <param name="resolution">The resolution</param>
+        /// <param name="selector">Selects a value from the BaseData to send into the indicator, if null defaults to the Value property of BaseData (x => x.Value)</param>
+        /// <returns>The T3MovingAverage indicator for the requested symbol over the specified period</returns>
+        public T3MovingAverage T3(Symbol symbol, int period, int volumeFactor, Resolution? resolution = null, Func<BaseData, decimal> selector = null)
```

Comment:
@mchandschuh If volume factor is 1, T3 is equivalent to DEMA(DEMA(DEMA(x))).
Looking at other implementations, the value is a parameter for T3.